docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
```

Comment:
@gblaih can you remove this docker compose?

Reply:
removed.